packages/legend-application-studio/src/application/LegendStudioApplicationConfig.ts

```diff
@@ -103,6 +103,7 @@ export interface LegendStudioApplicationConfigurationData
   sdlc: { url: string; baseHeaders?: RequestHeaders };
   depot: { url: string };
   engine: { url: string; queryUrl?: string };
+  query: { url: string };
```

Comment:
query should not be a required input for config. 
Think about studio only needs `sdlc,depot, engine` to run. it does not need stand alone query.